.clang-format-ignore

```diff
@@ -0,0 +1,3 @@
+# patricia comes from the CAIDA/cc-common project and is formatted by them
+patricia.c
+include/patricia.h
```

Comment:
I'm thinking the swig generated files should also be excluded. Otherwise, re-generation without re-formatting ends results in a misleading diff.